pkg/pillar/zedcloud/authen.go

```diff
@@ -514,10 +516,25 @@ func VerifySigningCertChain(log *base.LogObject, certs []*zcert.ZCert) ([]byte,
 			}
 		}
 	}
-	log.Tracef("VerifySigningCertChain: success\n")
 	return sigCertBytes, nil
 }
 
+type certStatus uint8
+
+const (
+	allCertsAreValid certStatus = iota
+	aCertIsInfo
+	aCertIsWarn
+	aCertIsINVALID
+)
+
+var (
+	lastCertTimeCheck time.Time
+	CurCertStatus     = allCertsAreValid
```

Comment:
Why does this need to be a global variable?

Reply:
Best way I could think to expose a warning.  Also, a way to clear once any problems have been cleared.